ang/civicase/CaseOverview.js

```diff
@@ -81,9 +83,32 @@
     };
 
     /**
-     * Toggle show breakdown dropdown
+     * Checks if all statuses are disabled
+     *
+     * @return {Boolean}
+     */
+    $scope.isAllStatusesDisbabled = function () {
+      return _.filter($scope.caseStatuses, function (status) {
+        return !status.disabled;
+      }).length === 0;
+    };
+
+    /**
+     * Toggle status view
+     *
+     * @param {event} event object
+     * @param {Number} index of the case status
      */
-    $scope.showHideBreakdown = function () {
+    $scope.toggleStatusView = function ($event, index) {
+      $scope.caseStatuses[index + 1].disabled = !$scope.caseStatuses[index + 1].disabled;
```

Comment:
Instead of the calling the property `disabled`, can we name this `hidden`? because we are just showing/hiding the status, and not really disabling it.
And if we decide to change this, and all vars and functions should also change which have the disabled keyword in this context.
Please feel free to flag any concerns.